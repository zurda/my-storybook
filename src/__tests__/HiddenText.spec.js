import React from 'react'
import { create } from 'react-test-renderer'
import HiddenText from '../components/HiddenText'

describe('Button component', () => {
  test('Matches the snapshot', () => {
    const hiddenText = create(
      <HiddenText
        filmArr={['T', 'o', 'y', ' ', 'S', 't', 'o', 'r', 'y']}
        guessedLetters={[' ', 'S', 's', 'T', 't']}
      />
    )
    expect(hiddenText.toJSON()).toMatchSnapshot()
  })
})
