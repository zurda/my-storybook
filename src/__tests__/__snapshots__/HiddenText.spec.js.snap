// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Matches the snapshot 1`] = `
<div
  className="css-nma54g"
>
  <span
    className="css-h5d4jr"
  >
    T
  </span>
  <span
    className="css-h5d4jr"
  >
    ?
  </span>
  <span
    className="css-h5d4jr"
  >
    ?
  </span>
  <span
    className="css-1ep591g"
  >
     
  </span>
  <span
    className="css-h5d4jr"
  >
    S
  </span>
  <span
    className="css-h5d4jr"
  >
    t
  </span>
  <span
    className="css-h5d4jr"
  >
    ?
  </span>
  <span
    className="css-h5d4jr"
  >
    ?
  </span>
  <span
    className="css-h5d4jr"
  >
    ?
  </span>
</div>
`;
